_docs/tasks/security/authn-policy.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Basic Istio Authentication Policy
+overview: Shows you how to use Istio authentication policy to setup mutual TLS and simple end-user authentication.
+
+order: 10
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+Through this task, you will learn how to:
+
+* Use authentication policy to setup mutual TLS.
+
+* Use authentication policy to do end-user authentication.
+
+
+## Before you begin
+
+* Understand Isio [authentication policy]({{home}}/docs/concepts/security/authn-policy.html) and related [mutual TLS authentication]({{home}}/docs/concepts/security/mutual-tls.html) concepts.
+
+* Know how to verify mTLS setup (recommend to walk through [testing Istio mutual TLS authentication]({{home}}/docs/tasks/security/mutual-tls.html))
+
+* Have a Kubernetes cluster with Istio installed, without mTLS. See [the Istio installation task]({{home}}/docs/setup/kubernetes/quick-start.html) and follow step 5a.
+
+* For demo, create two namespaces `foo` and `bar`, and deploy [httpbin](https://github.com/istio/istio/tree/master/samples/httpbin) and [sleep](https://github.com/istio/istio/tree/master/samples/sleep) with sidecar on both of them. Also, run another sleep app without sidecar (to keep it separate, run it in `legacy` namespace)
+
+```bash
```

Comment:
Indent this code block by 3 spaces so that it renders as part of the previous bullet item.

Reply:
Done